Components/Tests/ComponentsTests/WKSourceEditorFormatterTests.swift

```diff
@@ -766,4 +772,170 @@ final class WKSourceEditorFormatterTests: XCTestCase {
             XCTAssertEqual(base2Attributes[.font] as! UIFont, fonts.baseFont, "Incorrect base formatting")
             XCTAssertEqual(base2Attributes[.foregroundColor] as! UIColor, colors.baseForegroundColor, "Incorrect base formatting")
         }
+
+    func testSubscript() {
+        let string = "Testing. <sub>Subscript.</sub> Testing"
+        let mutAttributedString = NSMutableAttributedString(string: string)
+
+        for formatter in formatters {
+            formatter.addSyntaxHighlighting(to: mutAttributedString, in: NSRange(location: 0, length: string.count))
+        }
+
+        var base1Range = NSRange(location: 0, length: 0)
+        let base1Attributes = mutAttributedString.attributes(at: 0, effectiveRange: &base1Range)
+
+        var subscriptOpenRange = NSRange(location: 0, length: 0)
+        let subscriptOpenAttributes = mutAttributedString.attributes(at: 9, effectiveRange: &subscriptOpenRange)
+
+        var subscriptContentRange = NSRange(location: 0, length: 0)
+        let subscriptContentAttributes = mutAttributedString.attributes(at: 14, effectiveRange: &subscriptContentRange)
+
+        var subscriptCloseRange = NSRange(location: 0, length: 0)
+        let subscriptCloseAttributes = mutAttributedString.attributes(at: 27, effectiveRange: &subscriptCloseRange)
```

Comment:
Make this 24, to match what you have at line 818. I don't expect it to change the test result, but this ensures we're checking attributes at the exact same index. The attributes are identical at both index 27 and 24, which is why your tests pass fine even with this discrepancy.